dev/content/notebooks/Random Variable Algebra.ipynb

```diff
@@ -0,0 +1,597 @@
+{
```

Comment:
<p>nit: "to vaccinate"</p><p> </p><p>other nit: "random variable right right" there's an extra "right" here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/canyon289/canyon289.github.io/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>